frontend/src/mocks/handlers.ts

```diff
@@ -187,6 +187,10 @@ export const handlers = [
         ENABLE_BILLING: false,
         HIDE_LLM_SETTINGS: mockSaas,
       },
+      // Uncomment the following to test the maintenance banner
+      // MAINTENANCE: {
+      //   startTime: "2024-01-15T10:00:00-05:00", // EST timestamp
+      // },
```

Comment:
Should we make this controlled by an env var, so we don't need to change the code every time we do maintenance?

Reply:
Got it, thanks.